src/common/types.ts

```diff
@@ -144,6 +144,23 @@ export interface IPluginLookup {
   [pluginName: string]: IPlugin;
 }
 
+export class CChannelInfo {
+  id: string;
```

Comment:
What's the point in this class when you can get all this info straight from the GuildChannel class?

Reply:
So I can package up the info and send it to a function and not have any majog numbers.  Otherwise not sure on how I would do it as you can't send random Jsons through function in typeScript as far as I know.

Reply:
You could just send the whole `GuildChannel`

Reply:
I could have used 'this' after further inspection, however, I removed the class as I realized the channel id part was causing the catch to trigger so it wouldn't even show the error with that info.  So I just send the name of the channel now to the function now.